main.go

```diff
@@ -48,22 +53,36 @@ func main() {
 		return
 	}
 
-	if opts.NArgs() != 1 && opts.NArgs() != 2 {
+	if opts.NArgs() > 2 {
 		getopt.PrintUsage(os.Stderr)
 		os.Exit(1)
 		return
 	}
 
+	var target *url.URL
+	var err error
 	args := opts.Args()
-	target, err := url.Parse(args[0])
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to parse target '%s': %s\n", args[0], err)
-		os.Exit(1)
-		return
+	if len(args) == 0 {
+		target, err = url.Parse(os.Getenv("GOTCHA_BACKEND"))
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to parse target '%s': %s\n", os.Getenv("GOTCHA_BACKEND"), err)
+			os.Exit(1)
+			return
+		}
+	} else {
+		target, err = url.Parse(args[0])
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to parse target '%s': %s\n", args[0], err)
+			os.Exit(1)
+			return
+		}
```

Comment:
This is a lot of code duplication.  Can we set a target based on flags / environment, and only parse / check it the one time?